### PR TITLE
Added robots.txt

### PR DESCRIPTION
**Pull request by OleksSobol**
https://www.dev-eth0.de/blog/2016/11/13/jekyll_seo.html
https://sounoob.com.br/sitemap-e-robots-no-jekyll/
https://hashedin.com/blog/5-absolute-must-have-jekyll-plugins-if-you-care-about-performance-and-seo/

https://github.com/jekyll/jekyll-sitemap

### CODE REVIEW

**Comment by khusnetdinov** [comment]
@OleksSobol, Thanks for PR!. Just could you check if this version of the file is enough? I updated the description of the issue and added resources. For checking.
And I attach here screen of a compiled project `_site` for investigation if red squared files must be included to `robots.txt` files.
Alt text

**Comment by OleksSobol** [comment]
You want these files to be disallowed to the search engine?

**Comment by khusnetdinov** [comment]
@OleksSobol, Yes, as I think, this files must be disabled. You need to install Jekyll plugin, in future sitemap.xml will be created by this plugin. For this time you can use our stage domain `https://betterdocs.herokuapp.com/` it is working application. And it would be great if you check recommendations in attached links. Pay attention to permalinks and routes.

**Comment by OleksSobol** [comment]
I hope I did everything right! Please check!

File: _config.yml [hunk 1]
```diff
@@ -22,6 +22,10 @@ theme: minima
 plugins:
   - jekyll-feed
   - jekyll-multiple-languages-plugin
+  - jekyll-sitemap
+
+#Sitemap URL
+url: "https://betterdocs.herokuapp.com/"
```
[the remark this answers is not in the document]
Comment (khusnetdinov) [reply _config.yml]: @OleksSobol, After all, please squash 5 commits to 1 commit (One task - one commit )with the message: "[#80] Add robots.txt".

[the remark this answers is not in the document]
Comment (OleksSobol) [reply _config.yml]: Sorry about that! Just getting in the GitHub/Git World) Can you check it again)

**Comment by OleksSobol** [comment]
Thank you! The was fun! You are good mentor))

**Pull request merged**